### ISSUE TITLE
Formalise horizontal composition (Section 6.3 of RS17 paper)

### ISSUE THREAD

**Issue by fizruk** [rzk-lang/sHoTT#10]
Should be straightforward.

**Comment by fredrik-bakke** [comment]
That's right. You can find it here: https://rzk-lang.github.io/sHoTT/simplicial-hott/06-2cat-of-segal-types.rzk/#horizontal-composition

**Comment by fizruk** [comment]
My understanding is that this repository has a different goal: formalising as mush simplicial HoTT as we (including all the contributors) can, such as results by Jonathan Weinberger, Ulrich Buchholtz, and César Bardomiano Martínez. The Yoneda repository will still get updated with at least the cocartesian version of the lemma (when we have it), and, possibly, with formalisations in other proof assistants (or at least links to such formalisations and some comparisons). Perhaps, @emilyriehl can provide a more precise relationship of the two repositories.

**Comment by fizruk** [comment]
> I propose we close this issue and perhaps open a new one proving middle four interchange?

Sounds good to me :)
